todoist_api_python/authentication.py

```diff
@@ -43,9 +45,8 @@ def revoke_auth_token(
         "client_secret": client_secret,
         "access_token": token,
     }
-    response = post(session=session, url=endpoint, data=payload)
-
-    return response
+    post(session=session, url=endpoint, data=payload)
+    return True
```

Comment:
I don't see any changes to this. Why are we ignoring the response and always returning True for this and other endpoints? Are we sure this never fails?

Reply:
I think this was accidental, while I was testing something, or trying to fix some pyright error. Good catch. Resolved by: https://github.com/Doist/todoist-api-python/pull/61/commits/6f448d22df9ed370e83e4b14a95f65be757c9c92

Reply:
This is happening for most methond on [todoist_api_python/api.py](https://github.com/Doist/todoist-api-python/pull/61/files#diff-c9e0cccf5b7f01426dcdc94fe12598c1b8f1629de0a2014472479aee4fcbc702). Are you sure this is accidental?

Reply:
Hm, let me have a closer look, I experimented with the return type of [todoist_api_python/http_requests.py](https://github.com/Doist/todoist-api-python/blob/6f448d22df9ed370e83e4b14a95f65be757c9c92/todoist_api_python/http_requests.py), but I don't think I modified the return types of the API functions. This is weird.  Let me fix it.

Reply:
Should be resolved by https://github.com/Doist/todoist-api-python/pull/61/commits/21d8caff54e866f7351d1c489ee0ce190a0de543

Reply:
Thanks! Looks good now. I'm closing this comment and ask for more changes in a different one.